qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -14,38 +14,38 @@
 
 public class ManageAppointmentsPage extends Page {
 
-	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
+	public static final By DELETE_REQUEST_ICON = By.cssSelector(".delete-item.icon-remove");
 	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
+	private static final By SAVE_BUTTON = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
 	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons button");
-	private static final By SEARCH = By.cssSelector("#searchButtons button.confirm");
-	private static final By NEXT = By.cssSelector("#selectAppointment button.confirm");
-	private static final By BOOK_APPOINTMENT = By.cssSelector("i.icon-calendar:nth-child(1)");
-	private static final By YES_DELETE_REQUEST = By.xpath("//div[@id='confirm-cancel-appointment-request']/div[2]/button");
+	private static final By SEARCH_BUTTON = By.cssSelector("#searchButtons button.confirm");
+	private static final By NEXT_BUTTON = By.cssSelector("#selectAppointment button.confirm");
```

Comment:
I think we can have this renamed to `CONFIRM_BUTTON` instead of `NEXT_BUTTON`

Reply:
@sherrif10 , @kdaud  no it is not the final button that confirms the saving of an appointment, it is a NEXT button that when clicked leads to another stage where finally the save button is clicked to save the appointment. I still suggest that we maintain the NEXT_BUTTON as opposed to CONFIRM_BUTTON.

Reply:
Do you mind pointing us to the last confirm button url